compiler/rustc_passes/src/liveness.rs

```diff
@@ -1521,6 +1521,14 @@ impl<'tcx> Liveness<'_, 'tcx> {
     }
 
     fn warn_about_unused_args(&self, body: &hir::Body<'_>, entry_ln: LiveNode) {
+        if let Some(intrinsic) =
+            self.ir.tcx.intrinsic(self.ir.tcx.hir().body_owner_def_id(body.id()))
+        {
+            if intrinsic.must_be_overridden {
```

Comment:
Isn't this wrong? It seems like it will silence the warning even for intrinsics with a body.